Sources/LocalReceiptParsing/BasicTypes/AppleReceipt.swift

```diff
@@ -18,6 +18,24 @@ import Foundation
 /// - Seealso: [the official documentation](https://rev.cat/apple-receipt-fields).
 public struct AppleReceipt: Equatable {
 
+    /// The server environment a receipt belongs to.
+    public enum Environment: String {
+        /// Apps downloaded from the App Store
+        case production = "Production"
+
+        /// Development build or downloaded from TestFlight
+        case sandbox = "ProductionSandbox"
+
+        /// StoreKit Testing in Xcode
+        case xcode = "Xcode"
+
+        /// Unknown environment
+        case unknown = "Unknown"
```

Comment:
There is also `ProductionVPP` and `ProductionVPPSandbox` for Apple's Volume Purchase Program, but I believe those apps cannot have in-app purchases anyway.